src/WebJobs.Script.WebHost/WebScriptHostBuilderExtension.cs

```diff
@@ -91,6 +94,18 @@ public static IHostBuilder AddWebScriptHost(this IHostBuilder builder, IServiceP
                     loggingBuilder.AddWebJobsSystem<SystemLoggerProvider>();
                     if (environment.IsAzureMonitorEnabled())
                     {
+                        if (environment.IsConsumptionOnLegion())
+                        {
+                            loggingBuilder.Services.AddOptions<LoggerFilterOptions>()
+                            .Configure<IConfiguration>((options, config) =>
```

Comment:
nit: indentation. Lines 100-107 should be indented 1 level further. When splitting multi-line calls, the lines after the first should be indented 1 level further.
```suggestion
                                .Configure<IConfiguration>((options, config) =>
```

Reply:
Done